remoteit/desktop
# [DESK-463] One click registration

### Description


To support single click registration of the device and local services quick and easy.

### Checklist




### Test plan



1. Start up desktop with elevated permissions
2. Go to find.remote.it and find the system
3. Click connect
4. Login
5. You should land on the device setup screen
6. The hostname should be in the device name field
7. All local services should appear in a checked state
8. Click on the register button
9. Spinner should spin
10. Page should change and confirmation snackbar message should appear
11. After a minute, your device should appear in the device list

### Ticket



<https://remoteit.atlassian.net/browse/DESK-463>
